newa/cli.py

```diff
@@ -41,35 +41,55 @@
     )
 
 JIRA_NONE_ID = '_NO_ISSUE'
+STATEDIR_PARENT_DIR = '/var/tmp/newa'
+STATEDIR_NAME_PATTERN = '^run-([0-9]+)$'
```

Comment:
I'd recommend making this a raw string. There are no regex features that would be affected, but it's a good habit to build.

Reply:
fixed